src/accelerate/checkpointing.py

```diff
@@ -75,45 +76,49 @@ def save_accelerator_state(
             An optional gradient scaler instance to save
         save_on_each_node (`bool`, *optional*):
             Whether to save on every node, or only the main node.
+        safe_serialization (`bool`, *optional*, defaults to `True`):
+            Whether to save the model using `safetensors` or the traditional PyTorch way (that uses `pickle`).
     """
+    output_dir = Path(output_dir)
     # Model states
     for i, state in enumerate(model_states):
-        weights_name = f"{MODEL_NAME}.bin" if i == 0 else f"{MODEL_NAME}_{i}.bin"
-        output_model_file = os.path.join(output_dir, weights_name)
-        save(state, output_model_file, save_on_each_node=save_on_each_node)
+        weights_name = f"{MODEL_NAME}" if i == 0 else f"{MODEL_NAME}_{i}"
```

Comment:
In transformers, the model is named `pytorch_model` for bin format  and `model` for safetensors format. Is this something we want to do in accelerate ? In the `save_model` method, I use `WEIGHTS_NAME` and `SAFE_WEIGHTS_NAME` .

Reply:
We can change the default, and then have a warning for 3 issues if we find others that are using the old name and raise a deprecation

Reply:
Actually, I think we'll be okay doing this a bit cleaner without deprecation... :)